modules/nw-metallb-bgp-limitations.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-metallb.adoc
+
+[id="nw-metallb-bgp-limitations_{context}"]
+= Limitations for BGP mode
+
+[id="nw-metallb-bgp-limitations-break-connections_{context}"]
+== Node failure can break all active connections
+
+MetalLB shares a limitation that is common to BGP-based load balancing.
+When a BGP session terminates, such as when a node fails or when a `speaker` pod restarts, the session termination might result in resetting all active connections.
+End users can experience a `Connection reset by peer` message.
+
+The consequence of a terminated BGP session is implementation-specific for each router manufacturer.
+However, you can anticipate that a change in the number of `speaker` pods affects the number of BGP sessions and that active connections with BGP peers will break.
+
+To avoid or reduce the likelihood of a service interruption, you can specify a node selector when you add a BGP peer.
+By limiting the number of nodes that start BGP sessions, a fault on a node that does not have a BGP session has no affect on connections to the service.
+
+[id="nw-metallb-bgp-limitations-communities-values_{context}"]
+== Communities are specified as 16-bit values
+
+Communities are specified as part of an address pool custom resource and are specified as 16-bit values separated by a colon.
+For example, to specify that load balancer IP addresses are advertised with the well-known `NO_ADVERTISE` community attribute, use notation like the following:
+
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: AddressPool
+metadata:
+  name: doc-example-no-advertise
+  namespace: metallb-system
+spec:
+  protocol: bgp
+  addresses:
+    - 192.168.1.100-192.168.1.255
+  bgpAdvertisements:
+  - communities:
+    - 65535:65282
+----
+
+The limitation that communities are only specified as 16-bit values is a difference with the community-supported implementation of MetalLB that supports a `bgp-communities` field and readable names for BGP communities.
+
+[id="nw-metallb-bgp-limitations-single-asn_{context}"]
+== Support for a single ASN and a single router ID only
+
+When you add a BGP peer custom resource, you specify the `spec.myASN` field to identify the Autonomous System Number (ASN) that MetalLB belongs to.
+{product-title} uses an implementation of BGP with MetalLB that requires MetalLB to belong to a single ASN.
+If you attempt to add a BGP peer and specify a different value for `spec.myASN` than an existing BGP peer custom resource, you receive an error.
+
+Similarly, when you add a BGP peer custom resource, the `spec.routerID` field is optional.
+If you specify a value for this field, you must specify the same value for BGP peer custom resources that you add.
```

Comment:
@skrthomas Could you add bold word in the sentence above to add more clarity?
If you specify a value for this field, you must specify the same value for **all other** BGP peer custom resources that you add.